### PR TITLE
feat(patterns/breadcrumb): add breadcrumb pattern

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Add pattern Breadcrumb

GitHub issue number or Jira issue URL: [ID-1371](https://adeo.kanbanize.com/ctrl_board/10/cards/1371/details/)

### CODE REVIEW

File: src/docs/Components/Breadcrumb/code.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      <a href="#" class="mc-breadcrumb__link">Level 01</a>
+    </li>
+    <li class="mc-breadcrumb__item is-active">
+      <span class="mc-breadcrumb__link" aria-current="page">Level 02</span>
+    </li>
+  </ul>
+</nav>
+```
+
+<Preview path="responsive" />
```
Comment (nherwegh) [inline src/docs/Components/Breadcrumb/code.mdx:98]: Can we force the display of a `s-large` (for example) viewport size here?

File: src/docs/Components/Breadcrumb/code.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      <a href="#" class="mc-breadcrumb__link">Level 00</a>
+    </li>
+    <li class="mc-breadcrumb__item">
+      <span class="mc-breadcrumb__link" aria-current="page">Level 01</span>
+    </li>
+  </ul>
+</nav>
+```
+
+<Preview path="default" />
```
Comment (nherwegh) [inline src/docs/Components/Breadcrumb/code.mdx:42]: can't we use regular Mozaïc links to have the same behaviour (hover, focus, etc.)?

Comment (tiloyi) [reply src/docs/Components/Breadcrumb/code.mdx:54]: That's a very good point.
I just made the breadcrumb links use the style sheet of the "Links" pattern.

I'll let you check it out.

File: src/docs/Components/Breadcrumb/code.mdx (hunk 1 above)
Comment (tiloyi) [reply src/docs/Components/Breadcrumb/code.mdx:133]: Unfortunately, at the moment, we cannot force a preview to be displayed the first time it is loaded. But I'm writing it down to see how we can set this up for the next few times.

**Comment by tiloyi** [comment]
> @tiloyi breadcrumb icons are 14px square instead of 16px square

It's fixed.

**Pull request merged**